manifests/m/ManticoreGames/Core/1.3.0.0/ManticoreGames.Core.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+PackageIdentifier: ManticoreGames.Core
+PackageVersion: 1.3.0.0
+PackageLocale: en-US
+Publisher: Manticore Games
+PublisherUrl: https://www.coregames.com
+PrivacyUrl: https://www.coregames.com/legal/ManticorePrivacyPolicy.html
+PackageName: Core
+PackageUrl: https://www.coregames.com
+License: Copyright © 2020 Manticore Games, Inc. All Rights Reserved.
+LicenseUrl: https://www.coregames.com/legal/ManticoreLegalNotices.html
```

A copyright is not a license; Please update this
```suggestion
License: Proprietary
LicenseUrl: https://www.coregames.com/legal/ManticoreLegalNotices.html
Copyright: Copyright © 2020 Manticore Games, Inc. All Rights Reserved.
```